tools/package-unix.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+
+DIR="$(cd "$(dirname "${BASH_SOURCE[0]}")"/.. >/dev/null 2>&1 && pwd)"
+
+[ -z "$LLVM_DIR" ] && echo "Must set specific LLVM_DIR for packaging" && exit
+[ -z "$Clang_DIR" ] && echo "Must set specific Clang_DIR for packaging" && exit
+
+cmake -G Ninja -DCMAKE_BUILD_TYPE=Release \
```

Comment:
Another style it: IMHO this would be much easier to read if it was one-arg-per-line, preferably sorted, e.g.

```
cmake \
  -B "$DIR/build/static-Release" \
  -DBUILD_SHARED_LIBS=NO \
  -DClang_DIR="$Clang_DIR" \
  -DCMAKE_BUILD_TYPE=Release \
  -DHalide_BUNDLE_LLVM=YES \
  -DLLVM_DIR="$LLVM_DIR" \
  -DWITH_APPS=NO \
  -DWITH_DOCS=YES \
  -DWITH_PYTHON_BINDINGS=NO \
  -DWITH_TESTS=NO \
  -DWITH_TUTORIALS=NO \
  -DWITH_UTILS=NO \
  -G Ninja \
  -S "$DIR"
```

As it is now, trying to compare these two requires sub-scanning every line.

Reply:
Gonna wait for the build bots to finish up, then will just drive-by that since the buildbots don't run it.